README.md

```diff
@@ -184,33 +184,49 @@ This document serves as a comprehensive reference point for users, giving them i
 
 # Steps for execution
 
-Step 1:
-Install MongoDB using the following link:
+
+- Install MongoDB using the following link:
+
+- Install MongoDB using the following link:
 
 https://docs.mongodb.com/manual/tutorial/install-mongodb-on-windows
 
-Step 2:
-Git Clone the Repository
 
-    git clone https://github.com/VibhavDeo/FitnessApp.git
+- Git Clone the Repository
+
```

Comment:
The description is repeated.
It seems like that the README file is a bit messy.
Could you restore the README file and add the changes?